iothub/service/src/IotHubTokenCredentialProperties.cs

```diff
@@ -18,19 +18,20 @@ namespace Microsoft.Azure.Devices
     /// <summary>
     /// The properties required for authentication to IoT hub using a token credential.
     /// </summary>
-    internal class IotHubTokenCredential : IotHubCredential
+    internal class IotHubTokenCrendentialProperties
+        : IotHubConnectionProperties
     {
 #if NET451
 
-        public IotHubTokenCredential()
+        public IotHubTokenCrendentialProperties()
```

Comment:
ctor below fields

Reply:
ctor below both fields and properties?

Reply:
When I looked at the code, it is below the fields.

Reply:
I see 2 ctors, one for net451 and one for the others. I see fields between them.

Reply:
The fields are not for NET 451. I can add a separate #if NET451 block.

Reply:
I can make the update and you can see how it looks. The #if will be inverted though as there is no fields for NET451